Libraries/src/Amazon.Lambda.ApplicationLoadBalancerIdentity/README.md

```diff
@@ -0,0 +1,85 @@
+# Amazon.Lambda.ApplicationLoadBalancerIdentity
+
+This package contains Middleware that can be used in conjunction with ALB's OpenId Connect integration to populate the "User" property in the RequestContext.
+
+While it is in the Amazon.Lambda namespace, it can also be used in other ASP.NET Core deployment scenarios e.g. Fargate or EC2.
```

Comment:
Should we put this in its own repo so isn't so tied to Lambda. Fargate seems like an excellent use case and could be more common then Lambda.

Reply:
I'd be happy to do that, which repo should it move to?